Arrays.java

```diff
@@ -0,0 +1,69 @@
+public class Arrays {
+    public static void main(String[] args) {
+        int[] arrInt = new int[]{1, 2, 3};
```

Comment:
Здесь нужно было создать массив указанного размера и затем заполнить, вот так
```suggestion
        int[] arrInt = new int[3];
```
Текущий способ создания равносилен вот такому, т.е. без new
```java
        int[] arrInt = {1, 2, 3};
```